src/main/java/ch/uzh/ifi/hase/soprafs21/constant/ActivityCategory.java

```diff
@@ -1,18 +1,28 @@
 package ch.uzh.ifi.hase.soprafs21.constant;
 
 public enum ActivityCategory {
-    COOKING,
-    EATING,
-    THEATRE,
-    MOVIES,
-    MUSIC,
-    MUSEUMS,
-    OUTDOOR_ACTIVITY,
-    SIGHTSEEING,
-    TRAVEL,
-    WELLNESS,
-    SHOPPING,
-    GAMES,
-    SPORTS,
-    WINTERSPORTS
-}
+    COOKING("cooking-pot"),
+    EATING("hamburger"),
+    THEATRE("comedy"),
+    MOVIES("movie-projector"),
+    MUSIC("music"),
+    MUSEUMS("bust"),
+    OUTDOOR_ACTIVITY("campfire"),
+    SIGHTSEEING("camera"),
+    TRAVEL("airport"),
+    WELLNESS("spa-flower"),
+    SHOPPING("shopping-bag"),
+    GAMES("game-controller"),
+    SPORTS("track-and-field"),
+    WINTERSPORTS("skiing");
+
+    public final String icon;
+
+    ActivityCategory(String icon) {
+        this.icon = icon;
+    }
+
+    public String toString() {
+        return this.icon;
```

Comment:
This might change the representation of the enum in the database to be the icon name instead of the category.
I think it is not a huge problem - just something to think about.

Reply:
Maybe not overwriting the toString() method would be better, because if we dont overwrite it, calling toString() would return the Enum name (for example: ActivityCategory.GAMES.toString() would return "GAMES") and the representation in the database would not be the icon name, since we dont need the icon name as representation in the database

Reply:
@amoscalamida I'll take a look at it when using it in the ActivityService. I'll try not to depend on the values but on the keys. However extending the enum to an entity just to store the icon names at one place would be an overkill. 
(If something wouldn't work out we'll just return the keys and the front end would have to match them to the icons, providing a default icon for ActivityCategories that have newly been added.)

@AleksandarRistic99 I renamed it to `getIcon()`